test-utils/testData/api/parent.kt

```diff
@@ -17,219 +17,219 @@
 
 // TEST PROCESSOR: ParentProcessor
 // EXPECTED:
-// parent of File: B.java: null
-// parent of ITF: ITF
+// parent of YUV: <init>
+// parent of HSV: <init>
+// parent of String: <set-?>
+// parent of Anno: @Anno
```

Comment:
They are sorted per declaration container. I hope that minimizes the impact. What is your recommendation?

Reply:
what would the test be like without sorting them? If it is just ordering change it's easier to read just change the test data since you are changing the test data anyways?